docs/developer/architecture/centralized-cli.rst

```diff
@@ -0,0 +1,172 @@
+
+
+Cluster Administration
+^^^^^^^^^^^^^^^^^^^^^^
+
+**tool: mk8sctl**
+
++------------+------------+-----------------------------------------------+
+| Resource   | action     | parameters                                    |
++============+============+===============================================+
+| bootstrap  | deploy     |                                               |
++------------+------------+-----------------------------------------------+
+| cluster    | import-iso | path_to_iso                                   |
++------------+------------+-----------------------------------------------+
+| cluster    | upgrade    | dest-version, dry-run                         |
++------------+------------+-----------------------------------------------+
+| cluster    | downgrade  | dest-version, dry-run                         |
++------------+------------+-----------------------------------------------+
+| etcd       | health     |                                               |
++------------+------------+-----------------------------------------------+
+| bootstrap  | backup     |                                               |
++------------+------------+-----------------------------------------------+
+| bootstrap  | restore    | backup-file                                   |
++------------+------------+-----------------------------------------------+
+| cluster    | edit-conf  | conf-param=conf-value                         |
+|            |            | ex: ip-in-ip=true                             |
++------------+------------+-----------------------------------------------+
+
+Solution Administration
+^^^^^^^^^^^^^^^^^^^^^^^
+
+**tool: mk8sctl**
+
++------------+------------+-----------------------------------------------+
+| Resource   | action     | parameters                                    |
++============+============+===============================================+
+| solution   | import     |                                               |
++------------+------------+-----------------------------------------------+
+| solution   | import     | path_to_iso                                   |
++------------+------------+-----------------------------------------------+
+|environment | add        | name                                          |
++------------+------------+-----------------------------------------------+
+|environment | delete     |                                               |
++------------+------------+-----------------------------------------------+
+
+Cluster Service Administration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**tool: mk8sctl**
+
++------------+------------+-----------------------------------------------+
+| Resource   | action     | parameters                                    |
++============+============+===============================================+
+| grafana    | edit-conf  | conf-param=conf-value                         |
```

Comment:
This (and the ones below) won't work. If there's structured editing going on, then present the user with an editor.

Reply:
done